vulnerability-analyzer/src/main/resources/application.properties

```diff
@@ -83,6 +75,7 @@ scanner.ossindex.retry.multiplier=2
 scanner.ossindex.retry.max-attempts=6
 scanner.ossindex.retry.max-duration=2M
 scanner.ossindex.alias-sync-enabled=false
+scanner.ossindex.supported-package-url=cargo,composer,gem,hex,maven,npm,nuget,pypi,rpm,conan,conda,swift,cocoapods,cran
```

Comment:
Please also add the new properties to [`CONFIGURATION.md`](https://github.com/DependencyTrack/hyades/blob/main/CONFIGURATION.md) :)